cmd/kpod/images.go

```diff
@@ -155,7 +158,7 @@ func outputImages(store storage.Store, images []storage.Image, format string, ha
 			ID:        img.ID,
 			Name:      name,
 			Digest:    digest,
-			CreatedAt: createdTime,
+			CreatedAt: createdTime.Format("Jan 2, 2006 15:04"),
```

Comment:
I'll probably hate myself for forgetting after I get the answer, but why this date instead of the epoch?